python/tk_multi_workfiles/actions/file_action.py

```diff
@@ -124,15 +97,9 @@ def change_context(ctx):
         # restart engine:
         QtGui.QApplication.setOverrideCursor(QtCore.Qt.WaitCursor)
         try:
-            # If there's an engine running, change its context. Otherwise
-            # start the engine up in the new context.
-            if sgtk.platform.current_engine(): 
-                sgtk.platform.change_context(ctx)
-                return
-
-            sgtk.platform.start_engine(current_engine_name, ctx.sgtk, ctx)
+            sgtk.platform.change_context(ctx)
```

Comment:
we might need a min core constraint in info.yaml for this one.... maybe that's specified already?